code/obj/item/device/radios/headsets.dm

```diff
@@ -61,6 +61,21 @@
 	emp_act()
 		return //hax
 
+/obj/item/device/radio/headset/command/nt/commander
+	name = "\improper NT Commander's headset"
+	desc = "Issued to NanoTrasen Commanders, this radio headset can access several secure radio channels."
```

Comment:
i would put this next to the syndicate headset probably?

Reply:
I put it next to the command headsets as It's part of the command category like the NT one, no trouble moving it next to the Syndicate ones if that's more fitting

Reply:
It's right above the Syndicate headset now